### PR TITLE
Add itypes python package

### PR DESCRIPTION
**Pull request by k-dominik**
Hello,

this is my first recipe so I hope I did everything right. It builds allright locally.

My ultimate goal is to contribute a recipe for [apistar](https://pypi.python.org/pypi/apistar), and this is one of the missing dependencies.

### CODE REVIEW

File: recipes/itypes/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+test:
+  requires:
+    - flake8
+    - pytest
+  imports:
+    - itypes
+
+about:
+  home: https://github.com/tomchristie/itypes
+  license: BSD
```
Comment (CJ-Wright) [inline recipes/itypes/meta.yaml]: It would be good to have a licence_file (maybe open an issue) and a license_family

Comment (CJ-Wright) [reply recipes/itypes/meta.yaml]: Also 3 clause vs 2 clause spec would be good too

File: recipes/itypes/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+requirements:
+  build:
+    - python
+    - pip
+  run:
+    - python
+
+test:
+  requires:
```
Comment (CJ-Wright) [inline recipes/itypes/meta.yaml]: you don't need requirements if you don't run a test command.

**Comment by k-dominik** [comment]
Hey @CJ-Wright , thank you very much for your feedback! This is highly appreciated. I will raise issues in the related repos about licenses, will add those and will remove the requirements in the test section.

**Comment by k-dominik** [comment]
itypes is no longer a dependency of apistar, so I'm closing this

**Pull request closed without merging**